### PR TITLE
General UI fixes 3.7

### PR DESCRIPTION
**Pull request by Poppe-A**
#7556

### CODE REVIEW

File: libs/ui/src/lib/list/page/list-page.component.html [hunk 1]
```diff
@@ -59,7 +59,7 @@
         </div>
       </ng-container>
       <ng-template #displayList>
-        <div *ngFor="let item of items">
+        <div class="item-list" *ngFor="let item of items">
```
Comment (RemcoSimonides) [inline libs/ui/src/lib/list/page/list-page.component.html:62]: Let's call it `item` instead. Since this div is repeated for every item of items

**Review by RemcoSimonides — COMMENTED** [review]
Please ask Francois to review too since I cannot judge the changes in `styles/components/.scss`

File: libs/ui/src/lib/list/page/list-page.component.scss [hunk 2]
```diff
@@ -37,4 +37,8 @@ section {
 
 span.mat-body-2 {
   margin-bottom: 16px;
+}
+
+.item-list {
+  padding-bottom: 12px;
```
Comment (GrandSchtroumpf) [inline libs/ui/src/lib/list/page/list-page.component.scss:43]: shouldn't it be margin ? If you want to separate two elements, use margin

File: libs/event/src/lib/components/agenda-export/agenda-export.component.ts [hunk 3]
```diff
@@ -1,6 +1,7 @@
 import { Component, ChangeDetectionStrategy, Input } from '@angular/core';
 import { Event } from '@blockframes/event/+state/event.model';
 import { AgendaService } from '@blockframes/utils/agenda/agenda.service';
+import { boolean } from '@blockframes/utils/decorators/decorators';
```
Comment (RemcoSimonides) [inline libs/event/src/lib/components/agenda-export/agenda-export.component.ts:4]: Unused import

**Review by RemcoSimonides — APPROVED** [review]

**Review by GrandSchtroumpf — APPROVED** [review]

**Pull request merged**